src/core/InformationPanel/InformationPanelContent.tsx

```diff
@@ -0,0 +1,76 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import cx from 'classnames';
+import React from 'react';
+import { CommonProps, useTheme } from '../utils';
+import { useContainerWidth } from '../utils/hooks/useContainerWidth';
+import '@itwin/itwinui-css/css/information-panel.css';
+
+export type InformationPanelContentProps = {
+  /**
+   * Maximum width of the panel below which layout will shift from two columns to one,
+   * i.e. label and input will display on separate lines for widths lower than this value.
+   *
+   * If set to 0, the layout will always be two columns.
+   * If set to `Number.POSITIVE_INFINITY`, the layout will always be one column.
+   *
+   * @default 0
+   */
+  responsiveBreakpoint?: number;
+  /**
+   * Content of the component.
+   * Should be pairs of `Label` and input components.
```

Comment:
I dont think we should be that specific.. it should support anything in there with key, value lets say.. Or even more. Or our css is only handling those?

Reply:
Opinion?

Reply:
the grid already supports pairs of any two components, but using `Label` gives it some extra styling

Reply:
So if we put something else than Label, it should still be aligned? So I would suggest rephrasing the message from 'should' to 'could', for example..

Reply:
added the word "ideally"